src/SignalR/clients/csharp/Client/test/UnitTests/ServerSentEventsTransportTests.cs

```diff
@@ -359,7 +359,7 @@ public async Task SSETransportDoesNotSupportBinary()
                 var ex = await Assert.ThrowsAsync<ArgumentException>(() => sseTransport.StartAsync(new Uri("http://fakeuri.org"), TransferFormat.Binary).OrTimeout());
 
                 Assert.Equal("transferFormat", ex.ParamName);
-                Assert.Equal($"The 'Binary' transfer format is not supported by this transport.", ex.GetLocalizationSafeMessage());
+                Assert.Equal($"The 'Binary' transfer format is not supported by this transport. (Parameter 'transferFormat')", ex.GetLocalizationSafeMessage());
```

Comment:
Noooo, this is the wrong fix.

Change `GetLocalizationSafeMessage` to remove the "(Paramater" section of the string